assets/stylesheets/layout/_main.scss

```diff
@@ -375,10 +376,56 @@ $sidebar-width-min: 228px;
 	}
 }
 
+.focus-design {
+	.wp-primary {
+		opacity: 0.2;
+		pointer-events: none;
```

Comment:
Any reason why we're disabling pointer-events here?

It'd also be nice to stick to our [css guidelines](https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/css.md#csssass-coding-guidelines) here. This seems closer to being a modifer state for the page.